gameFunctions.js

```diff
@@ -261,11 +262,11 @@ const createGroup = (players, channelId) => {
             }, (err, res, body) => {
                 console.log(body);
                 if (err) {
-                    console.log(err);
+                    logger.logError(err);
                 }
             });
         }).catch(err => {
-            console.log(err);
+            logger.logError(err);
```

Comment:
How do we view these logs on Digital ocean, Can you make some documentation so we can access the droplet?

Reply:
I want to look at a viewer or something, but they also console log